api/watch/funcs_test.go

```diff
@@ -666,31 +709,22 @@ func TestConnectLeafWatch(t *testing.T) {
 
 func TestConnectProxyConfigWatch(t *testing.T) {
 	t.Parallel()
-	a := agent.NewTestAgent(t, t.Name(), `
-	connect {
-		enabled = true
-		proxy {
-			allow_managed_api_registration = true
```

Comment:
is this test still testing what it used to without this line?

Reply:
Yep, MakeTestClient defaults to connect on and managed api registrations allowed.